GliaWidgets/CallVisualizer/TemporaryCommandFile.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
```

Comment:
This file is also used in the secure conversations project right? So when we merge the secure conversations feature, we are going to have duplicates. Is that correct? If that's the case, then it would be good to create a ticket in the secure conversations project to address this, as that project will finish later than this one. What do you think?

Reply:
Yes, I agree. This file was taken from a secure conversation project to be applied to the visitor code. I think this file can be deleted once secure conversations is ready to be merged in future.

Reply:
or we can create PR with this file in master, and rebase feature branches as soon as PR will be merged into master.